query/src/provider.rs

```diff
@@ -198,6 +199,8 @@ impl<C: PartitionChunk + 'static> TableProvider for ChunkTableProvider<C> {
         filters: &[Expr],
         _limit: Option<usize>,
     ) -> std::result::Result<Arc<dyn ExecutionPlan>, DataFusionError> {
+        debug!("Input Filters to Scan: {:#?}", filters);
```

Comment:
```suggestion
        debug!(?filters, "Input Filters to Scan");
```

That way `filters` will be printing using `Debug` in its own logging field (rather than embedded in the log message)